site/docs/components/table.mdx

```diff
@@ -1130,7 +1238,465 @@ If your table has a large amount of data, you can use the `dense` property to fo
+      sortIconType: 'other',
+      transform: ({ age }) => {
+        return <div style={{ textAlign: 'right' }}>{age}</div>;
+      },
+    },
+    { key: 'profession', headerName: 'Profession' },
+  ];
+  const rows = [
+    { id: 1000, firstName: 'Lauri', surname: 'Kekkonen', age: 39, profession: 'Engineer' },
+    { id: 1001, firstName: 'Maria', surname: 'Sarasoja', age: 62, profession: 'Designer' },
+    { id: 1002, firstName: 'Anneli', surname: 'Routa', age: 50, profession: 'Meteorologist' },
+    { id: 1003, firstName: 'Osku', surname: 'Rausku', age: 18, profession: 'Mail Carrier' },
+    { id: 1004, firstName: 'Linda', surname: 'Koululainen', age: 8, profession: 'School student' },
+  ];
+  const caption = (
+    <span>
+      <b>Table 7</b>: Service users (sorting, dark variant)
+    </span>
+  );
+  return (
+    <div style={{ maxWidth: '640px' }}>
+      <Table
+        sortingEnabled
+        ariaLabelSortButtonUnset="Not sorted"
+        ariaLabelSortButtonAscending="Sorted in ascending order"
+        ariaLabelSortButtonDescending="Sorted in descending order"
+        indexKey="id"
+        renderIndexCol={false}
+        cols={cols}
+        rows={rows}
+        caption={caption}
+      />
+    </div>
+  );
+}}
+</Playground>
+
+#### Core code example:
+_Not supported in hds-core!_
+
+#### React code example:
+```tsx
+const cols = [
+  { key: 'id', headerName: 'Not rendered' },
+  { key: 'firstName', headerName: 'First name' },
+  { key: 'surname', headerName: 'Surname' },
+  {
+    key: 'age',
+    headerName: 'Age',
+    sortIconType: 'other',
+    transform: ({ age }) => {
+      return <div style={{ textAlign: 'right' }}>{age}</div>;
+    },
+  },
+  { key: 'profession', headerName: 'Profession' },
+];
+
+const rows = [
+  { id: 1000, firstName: 'Lauri', surname: 'Kekkonen', age: 39, profession: 'Engineer' },
+  { id: 1001, firstName: 'Maria', surname: 'Sarasoja', age: 62, profession: 'Designer' },
+  { id: 1002, firstName: 'Anneli', surname: 'Routa', age: 50, profession: 'Meteorologist' },
+  { id: 1003, firstName: 'Osku', surname: 'Rausku', age: 18, profession: 'Mail Carrier' },
+  { id: 1004, firstName: 'Linda', surname: 'Koululainen', age: 8, profession: 'School student' },
+];
+
+const caption = (
+  <span>
+    <b>Table 7</b>: Service users (sorting, dark variant)
+  </span>
+);
+
+return (
+  <div style={{ maxWidth: '640px' }}>
+    <Table
+      sortingEnabled
+      ariaLabelSortButtonUnset="Not sorted"
+      ariaLabelSortButtonAscending="Sorted in ascending order"
+      ariaLabelSortButtonDescending="Sorted in descending order"
+      indexKey="id"
+      renderIndexCol={false}
+      cols={cols}
+      rows={rows}
+      caption={caption}
+    />
+  </div>
+);
+```
+
+### Selecting rows
+HDS Table rows can be set to be selectable. This makes the first column to be a selection column. This also adds two (2) action buttons above the table; one button for selecting all rows and one button to clear selection.
```

Comment:
clear selection -> clear selections?

Reply:
I guess both are ok. Selection can include multiple elements.

Reply:
We can leave it as it is 👍